src/main/java/com/gistpetition/api/petition/dto/AgreementRequest.java

```diff
@@ -1,20 +1,20 @@
-package com.gistpetition.api.comment.dto;
+package com.gistpetition.api.petition.dto;
 
 
 import javax.validation.constraints.NotBlank;
 
-public class CommentRequest {
+public class AgreementRequest {
     @NotBlank
     private String content;
```

Comment:
```suggestion
    private String description;
```
DTO의 필드명과 Entitiy의 필드명을 같게 해주면 어떨까요?